### ISSUE TITLE
Theme -Parent StackOverflowException. - base example.

### ISSUE THREAD

**Issue by itfranck** [ironmansoftware/universal-dashboard#230]
Hi, using the 1.6.2 example from Gitbook, I am getting stackoverflow exception attempting to use a child theme.

```
$Theme3 = New-UDTheme -Name "Basic" -Definition @{
    UDDashboard = @{
        BackgroundColor = "rgb(255,255,255)"
        FontColor = "rgb(0, 0, 0)"
    }
  } -Parent "Azure"


$dash = New-UDDashboard -Title 'Debug' -Theme $Theme3 -Content {

}
$cert = ( Get-ChildItem -Path cert:\CurrentUser\My\69595c138b4dxxxxxxxxxxxxx8ef3f3dda )
Start-UDDashboard -Dashboard $dash -Certificate $cert -wait
```

POSHUD 1.6.2
IIS / Windows 10

**Comment by adamdriscoll** [comment]
Definitely a bug.

**Comment by adamdriscoll** [comment]
Fixed in 1.7.0
